manifests/init.pp

```diff
@@ -29,8 +29,10 @@
   $tomcat = $gutterball::params::tomcat,
 ) inherits gutterball::params {
 
+  $_ = cache_data('gutterball_db_password', $dbpassword)
+  $__ = cache_data('gutterball_db_user', $dbuser)
+  $___ = cache_data('gutterball_tomcat', $tomcat)
```

Comment:
@ehelms, @ekohl  i tried to without the assignment before. But I got an error message stating that cache_data required assignment.